ni_measurementlink_service/discovery/_client.py

```diff
@@ -111,6 +111,8 @@ def register_service(self, service_info: ServiceInfo, service_location: ServiceL
         Returns:
             ID that can be used to unregister the service.
         """
+        service_info.annotations["ni/service.programminglanguage"] = "Python"
```

Comment:
I'd like to see this constant centralized in an internal file so it can be reused here and in the test asserts.

This also applies to a couple others like 'ni\type_specialization'

We can discuss what's the best place.

Reply:
Maybe here
https://github.com/ni/measurementlink-python/tree/009b783fa139a17cc631b5d7be783b9604799dfe/ni_measurementlink_service/_internal/utilities

In a file like _constants.py. @bkeryan what do you think?

It could be like 'ANNOTATIONS_TYPE_SPECIALIZATION_KEY = 'ni/type_specialization'
etc.

Reply:
I don't feel strongly about whether we make constants for well-known annotations.

Reply:
I added constants

Reply:
Sorry, I overlooked the part about _internal/utilities. We should have deleted that module long ago.

Reply:
Deleted: https://github.com/ni/measurementlink-python/pull/604

Reply:
These constant names are very long.

Counter-proposal: ni_measurementlink_service/_annotations.py containing `TYPE_SPECIALIZATION_KEY`